bats/tests/helpers/vm.bash

```diff
@@ -289,6 +289,23 @@ launch_the_application() {
     fi
 }
 
+# Write a provisioning script that will be executed during VM startup.
+# Only a single script can be defined, and scripts are deleted by factory-reset.
```

Comment:
Should probably get a comment that the script needs to be provided by stdin and not as a parameter. Although it is kind of obvious once you look at the function.

Reply:
If possible, document that anyway — it's better to not need to look at any implementations at all when reading function documentation.